Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10 as build-python
+FROM python:3.8 as build-python
```

Comment:
Why downgrade this? I recommend against it. It could be considered a major breaking change for anyone customizing our app. If the concern is testing against minimal supported versions, let CI handle that.